server.js

```diff
@@ -87,11 +94,15 @@ if (useCookieSessionStore === 'true') {
 // Authentication middleware must be loaded before other middleware such as
 // static assets to prevent unauthorised access
 middleware.forEach(func => app.use(func))
+app.get('/', (req, res) => {
+  res.send('GOV.UK Prototype Kit (temporary home page)')
+})
 
 // Set up App
 var appViews = extensions.getAppViews([
   path.join(projectDir, '/app/views/'),
-  path.join(projectDir, '/lib/')
+  path.join(projectDir, '/lib/'), // TODO: Remove for v13
+  path.join(packageDir, '/lib/nunjucks') // TODO: Remove for v13
```

Comment:
We should be able to remove these lines now we are testing the kit as a package I think?

```suggestion
```